src/network/networkbase.h

```diff
@@ -0,0 +1,173 @@
+#ifndef SERVERDATA_H
+#define SERVERDATA_H
+
+#include <QObject>
+#include <QString>
+#include <QDateTime>
+#include <QJsonArray>
+#include <QJsonObject>
+#include <QJsonDocument>
+#include <QUdpSocket>
+#include <QTcpServer>
+
+#define CUSTOM_DEBUG
+
+// Custom types
+typedef quint16 PortNumType;
+typedef qint64  SizeType;
+
+// Default values for common parameters
+const QString     gDefaultAppLabel         = "PairStorm";
+const QString     gDefaultOrgLabel         = "cpp-lv425";
+
+const PortNumType gDefaultTcpPortNumber    = 36108;
+const PortNumType gDefaultUdpPortNumber    = 32807;
```

Comment:
not good way set to const ports numbers

Reply:
I have no choice by now. How we will move this information to the QSettings at future